dependency-check-suppressions.xml

```diff
@@ -4,43 +4,20 @@
   <!-- server related, mostly applicable to linux. hopefully get better cover with spring boot 2 -->
 
   <suppress>
-    <notes>
-      <![CDATA[
-## CVE-2016-5425
-
-The Tomcat package on Red Hat Enterprise Linux (RHEL) 7, Fedora, CentOS, Oracle Linux, and possibly other Linux distributions uses weak permissions for /usr/lib/tmpfiles.d/tomcat.conf, which allows local users to gain root privileges by leveraging membership in the tomcat group.
-
-## CVE-2016-6325
-
-The Tomcat package on Red Hat Enterprise Linux (RHEL) 5 through 7, JBoss Web Server 3.0, and JBoss EWS 2 uses weak permissions for (1) /etc/sysconfig/tomcat and (2) /etc/tomcat/tomcat.conf, which allows local users to gain privileges by leveraging membership in the tomcat group.
-
-## CVE-2017-6056
-
-Programming error in the processing of HTTPS requests in the Apache Tomcat servlet and JSP engine may result in denial of service via an infinite loop. The denial of service is easily achievable as a consequence of backporting a CVE-2016-6816 fix but not backporting the fix for Tomcat bug 57544. Distributions affected by this backporting issue include Debian (before 7.0.56-3+deb8u8 and 8.0.14-1+deb8u7 in jessie) and Ubuntu.
-
-## CVE-2018-1304
-
-The URL pattern of "" (the empty string) which exactly maps to the context root was not correctly handled in Apache Tomcat 9.0.0.M1 to 9.0.4, 8.5.0 to 8.5.27, 8.0.0.RC1 to 8.0.49 and 7.0.0 to 7.0.84 when used as part of a security constraint definition. This caused the constraint to be ignored. It was, therefore, possible for unauthorised users to gain access to web application resources that should have been protected. Only security constraints with a URL pattern of the empty string were affected.
-
-## CVE-2018-1305
-
-Security constraints defined by annotations of Servlets in Apache Tomcat 9.0.0.M1 to 9.0.4, 8.5.0 to 8.5.27, 8.0.0.RC1 to 8.0.49 and 7.0.0 to 7.0.84 were only applied once a Servlet had been loaded. Because security constraints defined in this way apply to the URL pattern and any URLs below that point, it was possible - depending on the order Servlets were loaded - for some security constraints not to be applied. This could have exposed resources to users who were not authorised to access them.
-]]>
-    </notes>
-    <gav regex="true">org\.apache\.tomcat:tomcat-annotations-api:8\.5\.27</gav>
+    <notes><![CDATA[ Spring Boot dependencies ]]></notes>
+    <gav regex="true">^org\.apache\.tomcat:tomcat-annotations-api:8\.5\.31*$</gav>
```

Comment:
regex ends at `31`. i think it's just `\d+` until it is solved with future version